src/lib/FreeC/Backend/Coq/Syntax.hs

```diff
@@ -223,3 +224,8 @@ requireImportFrom library modules = ModuleSentence
 requireExportFrom :: ModuleIdent -> [ModuleIdent] -> Sentence
 requireExportFrom library modules = ModuleSentence
   (Require (Just library) (Just Export) (NonEmpty.fromList modules))
+
+-- | Creates a @From ... Require ...@ sentence.
```

Comment:
To symbolize missing code we usually use `…` instead of `...` to explicitly communicate that the ellipses are not part of the languages.
```suggestion
-- | Creates a @From … Require …@ sentence.
```